src/fmeval/transforms/util.py

```diff
@@ -60,3 +60,32 @@ def validate_added_keys(current_keys: Set[str], original_keys: Set[str], keys_to
         f"and the original keys: {original_keys} does not match "
         f"the expected keys to be added: {keys_to_add}.",
     )
+
+
+def validate_call(call_method: Callable) -> Callable:
```

Comment:
Question: Since this works with transforms that mutate their input record, are there any plans to add a similar decorator for transforms that return a new record? 

Could we have some argument that's passed into the decorator to tell if the returned object is the mutated input record or a new record and that would change what `validate_call` would check for output?

Reply:
I'll consider adding this in a future PR. For now, I don't plan on implementing any transforms that return a new record, so we internally won't be needing this feature. Users can certainly implement their own decorator that does validation as they see fit, if they return a new record. Thanks for raising this point though; it's certainly valid, and it reminded me that I should include some examples of Transforms that return a new record in the documentation.